### PR TITLE
fix: clean-up tasks after the section merge

### PR DESCRIPTION
**Pull request by beyackle**
## Description

This solves a few lingering issues I noticed after we merged the adapter settings:

- The "Create a new bot" E2E test needs to be rewritten to handle the new creation experience, so I've put in a `.skip` temporarily that should be removed once we've done that.
- My test schemas were still in the server code, so those should go away.
- Another l10n sync

## Task Item

#minor

### CODE REVIEW

File: Composer/packages/client/src/pages/botProject/adapters/ExternalAdapterSettings.tsx [hunk 1]
```diff
@@ -53,9 +53,13 @@ const AdapterSettings = (props: Props) => {
 
   const externalServices = (schemas: (JSONSchema7 & { key: string })[]) => (
     <div>
-      <div css={subtitle}>
+      <div key={'subtitle'} css={subtitle}>
```
Comment (beyackle) [inline Composer/packages/client/src/pages/botProject/adapters/ExternalAdapterSettings.tsx:56]: Just a note: this seemingly useless change fixes a console warning.

File: Composer/cypress/integration/CreateNewBot.spec.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-context('Creating a new bot', () => {
+context.skip('Creating a new bot', () => {
```
Comment (cwhitten) [inline Composer/cypress/integration/CreateNewBot.spec.ts:4]: we shouldn't skip this

Comment (beyackle) [reply Composer/cypress/integration/CreateNewBot.spec.ts:4]: We shouldn't, but these E2E tests need to be re-written to cover the new creation dialog (which also pulls in adapters under the same feature flag).

Comment (cwhitten) [reply Composer/cypress/integration/CreateNewBot.spec.ts:4]: Can we turn off the new creation dialog when this is run?

Comment (beyackle) [reply Composer/cypress/integration/CreateNewBot.spec.ts:4]: I'll write some new E2E tests to replace these - there's not a lot here in the first place.

Comment (beyackle) [reply Composer/cypress/integration/CreateNewBot.spec.ts:4]: Actually, turning off the dialog is a better idea. I'll do that.

File: Dockerfile [hunk 3]
```diff
@@ -21,7 +21,7 @@ COPY ./Composer .
 COPY ./extensions ../extensions
 # run yarn install as a distinct layer
 RUN yarn install --frozen-lock-file $YARN_ARGS
-ENV NODE_OPTIONS "--max-old-space-size=4096"
+ENV NODE_OPTIONS "--max-old-space-size=10240"
```
Comment (carlosscastro) [inline Dockerfile:24]: Do we need this?

**Review by carlosscastro — APPROVED** [review]

**Comment by carlosscastro** [comment]
Looks good from the adapters functionality point of view. I'll let composer team comment on the docker and the CreateNewbot skip aspects.

File: Dockerfile (hunk 3 above)
Comment (beyackle) [reply Dockerfile:24]: This is what was making the "Docker Build verification" step keep failing in the automated steps. I don't know if 10240 is the right number, but we can decrease it later if we need to.

**Pull request merged**